app/assets/javascripts/search.js

```diff
@@ -31,7 +31,7 @@
       return searchResultData;
     },
     enableLiveSearchCheckbox: function ($searchResults) {
-      if ($searchResults.length > 0) {
+      if ($searchResults.length >= 0) {
```

Comment:
Is there ever a case where this could be negative?  If not, then this conditional seems to be redundant.